app/javascript/packs/admin/users/filtersModal.js

```diff
@@ -4,6 +4,67 @@ import {
   WINDOW_MODAL_ID,
 } from '@utilities/showModal';
 
+const updateDateRangeFilterIndicator = ({ startDate, endDate, indicator }) => {
+  const hasValues = startDate || endDate;
+  if (hasValues) {
+    indicator.classList.remove('hidden');
+  } else {
+    indicator.classList.add('hidden');
+  }
+};
+
+/**
+ * Some sections require the Preact DateRangePicker.
+ * This function imports the required packages and adds the pickers to the modal.
+ */
+const initializeDateRangePickers = async () => {
+  const joiningRangeContainer = document.querySelector(
+    `#${WINDOW_MODAL_ID} .js-joining-date-range`,
+  );
+
+  const joiningDateIndicator = document.querySelector(
+    `#${WINDOW_MODAL_ID} .js-filtered-indicator-joining-date`,
+  );
+
+  const [
```

Comment:
We're dynamically importing these since if a user never opens the filters modal then we don't actually need them 🤷‍♀️